packages/cms-data-sync/src/calendars/calendars.data-migration.ts

```diff
@@ -10,13 +11,16 @@ type CalendarItem = NonNullable<
   FetchCalendarsQuery['queryCalendarsContents']
 >[number];
 
+const limiter = new RateLimiter({ tokensPerInterval: 7, interval: 'second' });
```

Comment:
As the max requests are 7 do we want to give ourselves a buffer and set `tokensPerInterval` to 6 or 5?